src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// DragAndDrop controls the drag objet and data.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class DragAndDrop : BaseHandle
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public DragAndDrop() : this(Interop.DragAndDrop.New(), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        internal DragAndDrop(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public delegate void DragAndDropEventHandler(DragEvent dragEvent);
```

Comment:
이벤트를 받은 대상이 누구인지 명시해주면 좀 더 사용하기 편할것 같습니다.
```suggestion
        public delegate void DragAndDropEventHandler(View sender, DragEvent dragEvent);
```

Reply:
수정하였습니다.